assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/Chart.yaml

```diff
@@ -14,5 +14,4 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.0
-
+version: 0.1.1
```

Comment:
We should update this version on every update right?

Reply:
If we would maintain the chart independently, then yes. Right now, it's not a big deal if we don't update it.

Reply:
Does it not matter when helm does upgrades?

Reply:
Nope. Helm will evaluate it regardless.